metis-dereference/metis-dereference-service/src/main/java/eu/europeana/metis/dereference/service/MongoDereferenceService.java

```diff
@@ -145,41 +129,10 @@ public EnrichmentResultList dereference(String resourceId)
       result.put(resourceId, resource.getLeft());
     }
 
-    // Get links from dereferenced objects and try to enrich them. Get: 1) entities that have any of
-    // the dereferenced results as sameAs link, 2) entities that are mentioned as sameAs in any of
-    // the dereferenced results, and 3) entities that have a sameAs link that is in turn mentioned
-    // as sameAs link in any of the dereferenced results. So: transitivity of sameAs relations.
-    final Set<String> enrichmentLinks = new HashSet<>();
```

Comment:
The logic that follows sameAs or exactMatch links has been moved to the client. This is necessary because only there is the whole context available in the form of the full record.